src/System.Diagnostics.TraceSource/src/System/Diagnostics/DefaultTraceListener.cs

```diff
@@ -55,6 +82,13 @@ public override void Fail(string message, string detailMessage)
             WriteAssert(String.Empty, message, detailMessage);
         }
 
+         private void InitializeSettings() {
```

Comment:
>{ [](start = 43, length = 1)

Nit. Our coding style ([https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md)) is to put braces on their own line.